src/index.js

```diff
@@ -42,20 +47,25 @@ server.listen(3000, () => {
   console.log('Express server is ready.');
 });
 
-fs.readdirSync('./src/events/').forEach((file) => {
+readdirSync('./src/events/').forEach((file) => {
   const event = require(`../src/events/${file}`);
-  let eventName = file.split('.')[0];
+  const eventName = file.split('.')[0];
+
   console.log(`[ Event ] Loaded ${eventName}`);
-  client.on(eventName, event.bind(null, client));
+  if (event.once) {
+    client.once(event.name, (...args) => event.execute(...args));
+  } else {
```

Comment:
Codacy has a fix for the issue: [Closing curly brace appears on the same line as the subsequent block.](https://app.codacy.com/gh/Shadeslayer10/PokeRealm/pullRequest?prid=9275083)
```suggestion
  }
```